go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/btcsuite/btcwallet/wallet/txrules v1.0.0
 	github.com/btcsuite/btcwallet/walletdb v1.3.3
 	github.com/btcsuite/btcwallet/wtxmgr v1.2.0
+	github.com/btcsuite/fastsha256 v0.0.0-20160815193821-637e65642941
```

Comment:
IIRC we worked to remove this as there was a build error on a certain platform, cc @guggero?

Reply:
Ok, I was already wondering why that came up from nowhere

Reply:
We already used it in master, but it wasn't in `go.mod`. Go modules remains mysterious.

Reply:
Removed now